lib/matplotlib/tests/test_backend_svg.py

```diff
@@ -56,8 +56,7 @@ def test_noscale():
 
     fig = plt.figure()
     ax = fig.add_subplot(1, 1, 1)
-    ax.imshow(Z, cmap='gray')
-    plt.rcParams['svg.image_noscale'] = True
+    ax.imshow(Z, cmap='gray', interpolation='none')
```

Comment:
Given that this test is explicitly about the 'svg.image_noscale` option, perhaps we should make sure that the original code still works. I still have some qualms with the usage change here (although, it isn't like this rcParam has been around for all that long...).